src/ghdcbot/adapters/storage/sqlite.py

```
@@ -950,6 +965,162 @@ def update_social_profile_display(
+
+    def update_pr_channel_message(
+        self,
+        repo: str,
+        pr_number: int,
+        channel_id: str,
+        status: str | None = None,
+        events: list[dict] | None = None,
+        pr_title: str | None = None,
+    ) -> bool:
+        """Update status, events timeline, or title for a PR channel message."""
+        now = datetime.now(timezone.utc).isoformat()
```

Ruff (0.16.2)

[warning] Use `datetime.UTC` alias

Convert to `datetime.UTC` alias

(UP017)